tests/foreman/api/test_contentmanagement.py

```diff
@@ -480,42 +501,23 @@ def test_positive_capsule_sync(self):
         sync_status = capsule.content_get_sync()
         last_sync_time = sync_status['last_sync_time']
 
-        # If BZ1439691 is open, need to sync repo once more, as repodata
-        # will change on second attempt even with no changes in repo
-        if is_open('BZ:1439691'):
```

Comment:
I see this BZ didn't get enough traction and was CLOSED WONTFIX as stale. Can we really remove this entire branch? Shouldn't we rather make it unconditional, since behavior we considered a bug is just part of how Satellite works?

Reply:
Fixed.